### PR TITLE
chore: use new integration test runner instead of assert_ast() 

### PR DESCRIPTION
**Pull request by azjezz**
needs #126

### CODE REVIEW

File: src/parser/mod.rs [hunk 1]
```diff
@@ -388,7 +389,7 @@ impl Parser {
 
                     expect_token!("`=`", self, TokenKind::Equals);
 
-                    let value = self.expression(Precedence::Lowest)?;
```
Comment (azjezz) [inline src/parser/mod.rs:418]: this is a bug fix, declare value cannot be an expression, it needs to be a literal.

Comment (ryangjchandler) [reply src/parser/mod.rs:418]: yup, one of those cases where it was "get something working" :^)

**Comment by azjezz** [comment]
rebased.

**Comment by ryangjchandler** [comment]
still [WIP]?

**Comment by azjezz** [comment]
Yes, converting some more test :)

**Comment by azjezz** [comment]
@ryangjchandler I'm tired :D let's merge it as is, well do the rest another time ^^"

**Comment by ryangjchandler** [comment]
haha, no pressure at all. Appreciate the work so far!

**Pull request merged**